src/scripts/main.js

```diff
@@ -1,3 +1,15 @@
 'use strict';
 
-// write your code here
+const rows = document.querySelectorAll('tr');
+
+for (let i = 0; i < rows.length; i++) {
+  const row = document.querySelectorAll('tr')[i];
```

Comment:
You already have 'rows' variable, no need to querySelectorAll again. Replace 'document.querySelectorAll('tr')[i]' with 'rows[i]'.